composer.json

```
@@ -33,17 +33,23 @@
     ],
 
     "require": {
-        "php": ">=5.4.0",
+        "php": ">=7.1.0",
         "psr/log": "~1.0"
     },
 
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "^7.0"
```

The `php-7.1` requires PHPUnit `^7.0` version at least.

---

as long as it's still maintained. You could make at least 
```suggestion
        "phpunit/phpunit": "^8.0 || ^7.0"
```

So that travis picks the right version.

---

That's right. We can keep two versions supporting the current PHP versions.